ntbs-service/DataMigration/NotificationMapper.cs

```diff
@@ -121,6 +121,22 @@ private static Episode ExtractEpisodeDetails(dynamic rawNotification)
             };
         }
 
+        private static List<NotificationSite> AsSites(IEnumerable<dynamic> rawResultRawSites)
+        {
+            return rawResultRawSites
+                .Select(AsNotificationSite)
+                // Due to many->one mapping of legacy sites to ntbs sites, there might be clashes we need to deal with..
+                .GroupBy(site => site.SiteId)
+                .Select(clashingSites =>
+                {
+                    // .. to avoid data loss, we collect the freetext entered in all the possible members of the clash
+                    var combinedDescription = String.Join(", ", clashingSites.Select(site => site.SiteDescription));
+                    var canonicalSite = clashingSites.First();
+                    canonicalSite.SiteDescription = combinedDescription;
+                    return canonicalSite;
```

Comment:
I believe this will end up with the first site duplicated both as the primary mapped site, and again as part of the description.
SiteDescription is currently nvarchar(max) so no concerns for running into a length limit (should we introduce a length limit?)

Reply:
OK'ed in person - it's just the description that gets copied in